administration/scheduling-and-retries.md

```diff
@@ -10,6 +10,61 @@ Once an output plugin gets called to flush some data, after processing that data
 
 If the return status was **OK**, it means it was successfully able to process and flush the data, if it returned an **Error** status, means that an unrecoverable error happened and the engine should not try to flush that data again. If a **Retry** was requested, the _Engine_ will ask the _Scheduler_ to retry to flush that data, the Scheduler will decide how many seconds to wait before that happens.
 
+## Configuring Wait Time for Retry
+
+The Scheduler provides two configuration options called **scheduler.cap** and **scheduler.base** which can be set on service section. 
+
+| Key | Description | Default Value | 
+| -- | ------------| --------------| 
+| scheduler.cap | Set a maximum retry time in second. The property is supported from v1.8.7. | 2000 | 
+| scheduler.base | Set a base of exponential backoff. The property is supported from v1.8.7. | 5 |
+
+These two configuration determine the waiting time before retry happen. Fluent Bit use exponential-backoff-and-jitter algorithm to determine the waiting time. The waiting time is a random number. 
+
+For the Nth retry, the lower bound of the random number will be:
+
+`base`
+
+The upper bound will be:
+
+`min(base * (Nth power of 2), cap)`
+
+Given a example when base is setup to 3, cap is set up to 30. 
+
+1st time retry waiting time: The lower bound will be 3, the upper bound will be 3 * 2 = 6. So the waiting time will be a random number between (3, 6).
```

Comment:
I'd be tempted to remove the extra text in these bits so just "1st retry:..., 2nd retry:..."